storage/objects/stream_file_upload.go

```diff
@@ -43,8 +43,20 @@ func streamFileUpload(w io.Writer, bucket, object string) error {
 	ctx, cancel := context.WithTimeout(ctx, time.Second*50)
 	defer cancel()
 
+	o := client.Bucket(bucket).Object(object)
+
+	// Add preconditions so that the upload may safely retry on transient errors.
+	// In this case, we assume the object does not yet exist.
```

Comment:
I would change the line "In this case, we assume the object does not yet exist" to "The object does not yet exist in this code sample."